content/en/integrations/guide/azure-manual-setup.md

```diff
@@ -269,17 +269,44 @@ Datadog provides a recommended monitor you can use as a template to get started.
 
 After the integration tile is set up, metrics are collected by a crawler. To collect additional metrics, deploy the Datadog Agent to your VMs:
 
-#### Agent installation
+### Agent installation
 
-You can use the Azure extension to install the Datadog Agent on Windows VMs, Linux x64 VMs, and Linux ARM-based VMs.
+You can use the Azure extension to install the Datadog Agent on Windows VMs, Linux x64 VMs, and Linux ARM-based VMs. You can also use the AKS Cluster Extension to deploy the Agent to your AKS Clusters.
 
-1. In the [Azure portal][14], navigate to your **VM** > **Settings** > **Extensions** > **Add** and select **Datadog Agent**.
-2. Click **Create**, enter your [Datadog API key][15], and click **OK**.
+{{< tabs >}}
+{{% tab "VM Extension" %}}
+
+1. In the [Azure portal][4], select the appropriate VM.
+2. From the left sidebar, select **Extensions + applications** under **Settings**.
```

Comment:
I went back and forth on this. Non-blocking
```suggestion
2. From the left sidebar, under **Settings**, select **Extensions + applications**.
```

Reply:
Thanks @estherk15 , I agree this is better